R/add_activity_after_death_flag.R

```diff
@@ -11,19 +11,22 @@
 add_activity_after_death_flag <- function(
     data,
     year,
-    deaths_data = read_file(all_slf_deaths_lookup_path())) {
-  # Match on BOXI NRS deaths lookup for records without chi
-  data <- data %>%
+    deaths_data = read_file(get_all_slf_deaths_lookup_path())) {
```

Comment:
i think the `slfhelper::get_chi()` should go here 

deaths_data = read_file(get_all_slf_deaths_lookup_path()) %>% slfhelper::get_chi(())